assets/station/station_resources.tres

```diff
@@ -6,8 +6,7 @@
 script = ExtResource("1_o3hpo")
 id = 0
 hydrogen = 500
+max_hydrogen = 100
```

Comment:
Did this get deleted?

Reply:
i checked the git history but unless i missed it, seems like somehow it had not been added before